### DISCUSSION TITLE
Ability to duplicate built in workflows for projects

### DISCUSSION THREAD

**Discussion by IDrumsey** [community/community#103145] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

I know you can duplicate the the `auto add to project` workflow, but apparently this isn't currently possible for the `item added to project` workflow. I have a use case for this (so maybe this feature can be implemented?) - I want issues to have their status set to Todo whereas I want pull requests to be set to a custom status called `Review`. Currently I can't do this cuz I can only have one of those workflows.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by castarco** [comment]
> [!TIP]
> **Note:** There is a previous discussion about the same topic: #36584 , if you reach this comment and upvoted the current discussion, I suggest you to upvote the older one as well.

I totally agree. Besides the case you present, I have another one use case for that:
- I want that merging PRs moves the ticket to "Merged"
- But I want that closing an Issue moves the associated ticket to "Done/Deployed"
